src/app/docs/kagent/concepts/agents/page.mdx

```diff
@@ -47,6 +48,43 @@ Some tools support additional configuration that can be set in when adding the t
 
 Kagent comes with a set of built-in tools that you can use to interact with your environment. Kagent also supports the [MCP (Model Configuration Protocol)](https://modelcontextprotocol.io/introduction) tools. Using MCP, you can bring any external tool into kagent and make it available for your agents to run.
 
+## Skills
+
+Skills are descriptions of the capabilities that an agent has to act more autonomously. They make the LLM's responses more than just reactions to prompts by orienting them toward goals.
+
+In some frameworks, skills are expressed as wrapped functions, reusable prompt templates, or even a synonym for a tool.
+
+In kagent, think of skills like a catalog that expresses what the agent is capable of doing for a user. Unlike tools, skills are not a specific function that produces an output, like "fetch a website" or "tell the weather." Unlike system instructions, they are not rules that apply to all interactions, such as "Follow my company's style guide."
+
+Instead, skills are building blocks that guide the agent's tool usage and planning. They help the agent understand what its goals are, and when and how to use tools effectively.
+
+For example, two agents may share the same tools but use them differently based on their skills:
+
+- A troubleshooting agent might use a `describe` tool to check the events of a crashing pod before taking a recovery action, such as restarting the pod.
+- A research agent might use the same `describe` tool to gather details about a pod in order to answer a user's question.
+
+Kagent supports two types of skills:
+
+1. **A2A skills** (metadata): Structured descriptions of capabilities defined in `a2aConfig.skills`. These consist of a description, examples, ID, and tags that help guide the agent's behavior. The description and examples provide context for both humans and the agent itself, often incorporated into system instructions. The ID and tags help you manage skills, such as by making it simpler to compare and reuse them across agents.
```

Comment:
note that this is just metadata and it's not actually used anywhere during agent execution or anything. I think the "kagent supports two types of skills" should be re-worded to talk about the fact that there's a concept of a skill in A2A, but it's not in any way related to the Anthropic skills.